scalafix-tests/input/src/main/scala-2/test/AnchorPatches.scala

```diff
@@ -1,28 +0,0 @@
-/*
-rule = [RemoveUnused, ExplicitResultTypes]
-ExplicitResultTypes.memberKind = [Def]
-ExplicitResultTypes.memberVisibility = [Public]
-unsafeShortenNames = true
-ExplicitResultTypes.unsafeShortenNames = true
- */
-package test
-
-import scala.collection.mutable
-import scala.collection.immutable // scalafix:ok RemoveUnusedImports
-
-import scala.collection.mutable.{ // scalafix:ok RemoveUnusedImports
-  Map,
-  Set
-}
-
-import scala.collection.mutable.{
-  Map,
-  Set
-}
-
-import scala.concurrent.duration.Duration
-
-object AnchorPatches {
-  def d1s = Duration(1, "s")
-  def d2s = Duration(2, "s") // scalafix:ok ExplicitResultTypes
```

Comment:
rewrote the test to leverage RemoveUnused only, as ExplicitResultTypes is not available on Scala 3, thus the undetected rename